docs/designers-developers/faq.md

```diff
@@ -241,7 +241,7 @@ Here is a brief animation illustrating how to find and use the keyboard shortcut
 
 ## Is Gutenberg built on top of TinyMCE?
 
-No. [TinyMCE](https://www.tinymce.com/) is one of the best tools for enabling rich text on the web. In Gutenberg, TinyMCE does exactly that. Nearly every text field you'll find is augmented with TinyMCE for rich text. Whether it be text, lists, or even just a single caption, TinyMCE can be invoked on blocks for rich text enhancements.
```

Comment:
Another reference worth updating:

https://github.com/WordPress/gutenberg/blob/b488d83b9f0863d01be4b6c558e23506a6f87a4d/docs/designers-developers/developers/tutorials/block-tutorial/introducing-attributes-and-editable-fields.md#L121

Reply:
Oh, not sure why searching for `TinyMCE` didn't catch this. Must have overlooked. :)